bats/helper/1pk5col-ints-rm-pk-col.schema

```diff
@@ -1,39 +1,8 @@
-{
-  "columns": [
-    {
-      "tag": 1,
-      "name": "c1",
-      "kind": "int",
-      "is_part_of_pk": false,
-      "col_constraints": []
-    },
-    {
-      "tag": 2,
-      "name": "c2",
-      "kind": "int",
-      "is_part_of_pk": false,
-      "col_constraints": []
-    },
-    {
-      "tag": 3,
-      "name": "c3",
-      "kind": "int",
-      "is_part_of_pk": true,
-      "col_constraints": []
-    },
-    {
-      "tag": 4,
-      "name": "c4",
-      "kind": "int",
-      "is_part_of_pk": false,
-      "col_constraints": []
-    },
-    {
-      "tag": 5,
-      "name": "c5",
-      "kind": "int",
-      "is_part_of_pk": false,
-      "col_constraints": []
-    }
-  ]
-}
+CREATE TABLE `test` (
```

Comment:
Maybe these schema file should all be embedded as SQL in the tests now? It's way easier to grok the tests if they are not sitting out here as files.

You could just do:
```
dolt sql <<<SQL
<QUERY>
SQL
```